.pipelines/azure-pipeline-build.yml

```diff
@@ -475,6 +479,170 @@ jobs:
         GdnBreakGdnToolSemmle: true
         GdnBreakGdnToolSemmleSeverity: 'Warning'
 
+- job: Linux_CCP_Prometheus_Collector
```

Comment:
We will need to add this to our EV2 release files too if they want a prod image

Reply:
Yeah, I have a task for it [here](https://msazure.visualstudio.com/InfrastructureInsights/_workitems/edit/25621892) I plan on adding it in once this PR merges to main